server/service/client_orbit.go

```diff
@@ -0,0 +1,77 @@
+package service
```

Comment:
note: both `baseClient` and `deviceClient` are in files named `base_client` and `device_client`, but this is in `client_orbit`

feel free to chose any pattern and adjust accordingly, but I'd suggest we stick with just one